src/components/Container/index.tsx

```diff
@@ -36,7 +36,7 @@ const Container = () => {
 
 			<div className={s.content}>
 				<div className={s.header}>
-					<HeaderTest />
+					<Header />
 				</div>
 				<h2>Todays Highlights</h2>
 				<div className={s.widget}>
```

Comment:
Visibility component. visibility of what?

Reply:
done